scripts/trivia.js

```diff
@@ -3511,6 +3511,90 @@ addAdminCommand(["removeextanswers"], function (src, commandData, channel) {
     }
 }, "Use to remove questions (by ID number) to the list of questions that should have a longer answer time when asked.");
 
+addAdminCommand(["updatelist"], function (src, commandData, channel) {
+    if (commandData === undefined) { return; }
+    if (commandData.length === 0 && trivData.updateList.length === 0) {
```

Comment:
what's the significance of checking commandData === undefined and then commandData.length === 0? It looks like they're trying to do the same thing :x